packages/skin/src/routes/_index/component/alert-dialog/+page.marko

```diff
@@ -25,7 +25,7 @@
                 id="alert-dialog-0"
                 role="alertdialog"
             >
-                <div class="alert-dialog__window alert-dialog__window--fade">
+                <div class="alert-dialog__window alert-dialog__window--animate">
```

Comment:
This isn't going to be a breaking change, right? Are we just remove the fate in lieu of animate?

Reply:
No, fade is still supported, but we are defaulting to animate